src/datasets/iterable_dataset.py

```diff
@@ -877,6 +878,8 @@ def map(
             remove_columns (`Optional[List[str]]`, defaults to `None`): Remove a selection of columns while doing the mapping.
                 Columns will be removed before updating the examples with the output of `function`, i.e. if `function` is adding
                 columns with names in `remove_columns`, these columns will be kept.
+            features (`Optional[datasets.Features]`, defaults to `None`): Use a specific Features to store the cache file
+                instead of the automatically generated one.
```

Comment:
there's no notion of "cache" for iterable datasets
```suggestion
            features (`Optional[datasets.Features]`, defaults to `None`): Feature types of the resulting dataset.
```